tethys_gizmos/templates/tethys_gizmos/gizmos/job_row.html

```diff
@@ -14,22 +14,75 @@
     </div>
     {% else %}
-      {% include "tethys_gizmos/gizmos/job_status.html" %}
+    <div data-toggle="tooltip" title="{{ job_status }}" align="left">
+      {% if job_status == 'Various' or job_status == 'Various-Complete' %}
+        <div class="progress" style="margin-bottom: 0;">
+          {% if job_statuses.Aborted > 0 %}
+          <div class="progress-bar progress-bar progress-bar" role="progressbar" title="Aborted" aria-valuenow="{{ job_statuses.Aborted }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ job_statuses.Aborted }}%; background-color: #555555;">
+            <span class="sr-only">{{ job_statuses.Aborted }}% Aborted</span>
+          </div>
+          {% endif %}
+          {% if job_statuses.Error > 0 %}
+          <div class="progress-bar progress-bar-danger progress-bar" role="progressbar" title="Error" aria-valuenow="{{ job_statuses.Error }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ job_statuses.Error }}%">
+            <span class="sr-only">{{ job_statuses.Error }}% Error</span>
+          </div>
+          {% endif %}
+          {% if job_statuses.Completed > 0 %}
+          <div class="progress-bar progress-bar-success progress-bar" role="progressbar" title="Complete" aria-valuenow="{{ job_statuses.Completed }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ job_statuses.Completed }}%">
+            <span class="sr-only">{{ job_statuses.Completed }}% Complete</span>
+          </div>
+          {% endif %}
+          {% if job_statuses.Running > 0 %}
+          <div class="progress-bar progress-bar-info progress-bar-striped active" title="Running" role="progressbar" aria-valuenow="{{ job_statuses.Running }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ job_statuses.Running }}%">
+            <span class="sr-only">{{ job_statuses.Running }}% Running</span>
+          </div>
+          {% endif %}
+        </div>
+      {% else %}
+        <div class="status-indicator status-{{ job_status }}"></div>{{ job_status }}
+      {% endif %}
+    </div>
     {% endif %}
   </td>
-  {% if delete %}
-    <td>
-      <a class='btn btn-danger btn-xs btn-job-delete' href="javascript:void(0)" data-job-id="{{ job_id }}" >Delete</a>
-    </td>
-  {% endif %}
-  {% if show_resubmit_btn %}
-    <td>
-      <a class='btn btn-warning btn-xs btn-job-resubmit' href="javascript:void(0)" data-job-id="{{ job_id }}" >Resubmit</a>
-    </td>
-  {% endif %}
-  {% if show_log_btn %}
-    <td>
-      <a class='btn btn-info btn-xs btn-job-show-log' href="javascript:void(0)" data-job-id="{{ job_id }}" data-toggle="modal" data-target="#modal-dialog-jobs-table-show-log">Logs</a>
-    </td>
-  {% endif %}
 {% endif %}
+{% if show_actions %}
+  <td>
+    <div class="btn-group">
+      <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        Actions <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
```

Comment:
Can we right-align the dropdown menu with the button? Since it is wider than the button and in the far right column, it is possible that it will extend past the table when it is open.

Reply:
Good point. I'll see what I can do.